src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp

```diff
@@ -50,68 +52,124 @@
 				*Please note that loading real patient data into the system is strictly prohibited!
 			</p>
 		</c:if>
-		<div class="row">
-			<div class="col-sm-6 col-sm-offset-3">
-				<form:form method="post" id="fm1" cssClass="" commandName="${commandName}" htmlEscape="true">
-					<div class="form-group has-error">
-						<div class="help-block">
-							<form:errors path="*" cssClass="" id="status" element="span"/>
-						</div>
-					</div>
-					<div class="form-group">
-						<label for="username">Username</label>
-						<form:input cssClass="form-control" cssErrorClass="" id="username" size="25" tabindex="1"
-									accesskey="${userNameAccessKey}" path="username" autocomplete="false"
-									htmlEscape="true"/>
-					</div>
-					<div class="form-group">
-						<label for="password">Password</label>
-							<%--
-							NOTE: Certain browsers will offer the option of caching passwords for a user.  There is a non-standard attribute,
-							"autocomplete" that when set to "off" will tell certain browsers not to prompt to cache credentials.  For more
-							information, see the following web page:
-							http://www.geocities.com/technofundo/tech/web/ie_autocomplete.html
-							--%>
-						<form:password cssClass="form-control" cssErrorClass="" id="password" size="25" tabindex="2"
-									   path="password" accesskey="${passwordAccessKey}" htmlEscape="true"
-									   autocomplete="off"/>
-					</div>
-					<input name="submit" id="submit" class="btn btn-lg btn-primary btn-block" accesskey="l" value="Login"
-						   tabindex="4" type="submit"/>
-					<div style="text-align: right">
-						<a href="${casProperties.applicationUrl}forgot_password.jsp">Forgot Password?</a>
+		<c:choose>
+			<c:when test="${casProperties.localOrLDAPAuthentication}">
```

Comment:
This is basically a check if eureka.authHandlers is null or empty, correct? This should be an easy JSTL expression, no need to implement a method.